arb_os/arbsys.mini

```diff
@@ -65,12 +67,14 @@ import impure func decompressAddress(stream: ByteStream) -> option<(ByteStream,
 import func rlp_encodeUint(val: uint, ba: ByteArray, offset: uint) -> (ByteArray, uint);
 import func rlp_encodeAddress(addr: address, ba: ByteArray, offset: uint) -> (ByteArray, uint);
 
+import impure func pcTableForCode(code: ByteArray) -> option<Stack>;
+
 
 public impure func arbsys_txcall() {
     if let Some(topFrame) = evmCallStack_topFrame() {
         let calldata = evmCallFrame_getCalldata(topFrame);
         if (bytearray_size(calldata) < 4) {
-            evmOp_revert(0, 0);
+            evmOp_revert_knownCodePc(address(100), 0, 0, 0);
```

Comment:
It might be better for debugging to give each ArbSys revert it's own artificial PC so that they can be distinguished.

Reply:
This is done in the next commit.